husky_directory/static/css/structure.css

```diff
@@ -0,0 +1,66 @@
+/* Not in our normal CSS; custom */
```

Comment:
What is "our normal CSS"?

Reply:
I pulled in structure.css, the comment is pre-existing